exercises/concept/interest-is-interesting/.meta/exemplar.rs

```diff
@@ -0,0 +1,35 @@
+pub fn interest_rate(balance: f64) -> f64 {
+    if balance < 0.0 {
+        return 3.213;
+    }
+
+    if balance < 1000.0 {
+        return 0.5;
+    }
+
+    if balance < 5000.0 {
+        return 1.621;
+    }
+
+    return 2.475;
+}
+
+pub fn interest(balance: f64) -> f64 {
+    return balance * interest_rate(balance) / 100.0;
+}
+
+pub fn annual_balance_update(balance: f64) -> f64 {
+    return balance + interest(balance);
+}
+
+pub fn years_before_desired_balance(balance: f64, target_balance: f64) -> u8 {
+    let mut accumulating_balance = balance;
+    let mut years = 0;
```

Comment:
I checked to see if students know about the `mut` keyword. It is mentioned in the introduction of the lasagna exercise, but I think adding a hints makes sense.